core/src/main/kotlin/com/justai/jaicf/builder/ScenarioBuilder.kt

```diff
@@ -276,7 +310,20 @@ abstract class ScenarioBuilder(
         fun intent(intent: String) = add(Transition(
             fromState,
             toState,
-            IntentActivationRule(intent)
+            IntentByNameActivationRule(intent)
+        ))
+
+        /**
+         * Appends any-intent activator to this state. Means that any intent can activate this state.
+         * Requires a [com.justai.jaicf.activator.intent.IntentActivator] in the activators' list of your [com.justai.jaicf.api.BotApi] instance.
+         *
+         * @see com.justai.jaicf.activator.intent.IntentActivator
+         * @see com.justai.jaicf.api.BotApi
+         */
+        internal fun anyIntent() = add(Transition(
```

Comment:
Are you sure it **should** be internal? Let's think about how the user could use this activation in other cases if they would like to activate some state that is already bound to any other activators.